cluster/Dockerfile.gpu

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+FROM nvidia/cuda:9.0-cudnn7-runtime-ubuntu16.04
```

Comment:
so does this GPU file actually work atm?

Reply:
I think it does, oddly enough. I built this from my desktop, and it appears to work.

I think it might *not* work if I build it from my macbook. More studies need to be done.